otx/algorithms/common/adapters/mmcv/hooks.py

```diff
@@ -567,3 +568,78 @@ def after_train_iter(self, runner: BaseRunner):
         if isnan(runner.outputs["loss"].item()):
             logger.warning("Early Stopping since loss is NaN")
             runner.should_stop = True
+
+
+@HOOKS.register_module()
+class EMAMomentumUpdateHook(Hook):
```

Comment:
Putting all the hook classes in hooks.py seems to make the file lengthy and uncomfortable to look at. How about creating a subfolder called `adapters/mmcv/hook` to separate them?

Reply:
I agree. I think we have to consider the situation that additional hooks (or other adapters) will be saved.